src/firefly/html/demo/ffapi-highlevel-test.html

```diff
@@ -23,6 +23,14 @@
     <br>
 </div>
 
+<div>
+    <!--<a href='javascript:firefly.getViewer("testChannel").showImage({url:"http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits"})'>test open</a>-->
+    <a href='javascript:firefly.getViewer().plotURL("http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits")'>test open</a>
```

Comment:
It is suppose to show a new one.  the other does only replaces.  I will look at the zoom level.